pkg/controllers/queue/queue_controller_action.go

```diff
@@ -173,3 +191,56 @@ func (c *queuecontroller) closeQueue(queue *schedulingv1beta1.Queue, updateState
 
 	return nil
 }
+
+func (c *queuecontroller) openHierarchicalQueue(queue *schedulingv1beta1.Queue) (bool, error) {
```

Comment:
The function of this function is more like checking. Its name represents an action. I think the name should be changed.
Or is there any functionality missing here?
If a queue is open, it should open its childqueue if the childqueue is closed because the parent queue is closed.